src/buckets/components/CreateBucketOverlay.tsx

```diff
@@ -24,19 +27,15 @@ import {
 // Selectors
 import {getOrg} from 'src/organizations/selectors'
 
-interface OwnProps {
-  onClose: () => void
-}
-
 type ReduxProps = ConnectedProps<typeof connector>
-type Props = OwnProps & ReduxProps
+type Props = ReduxProps
 
 const CreateBucketOverlay: FC<Props> = ({
   org,
   isRetentionLimitEnforced,
   createBucket,
-  onClose,
 }) => {
+  const {onClose} = useContext(OverlayContext)
```

Comment:
is this available for the existing paradigm?

Reply:
Yop yop yop